fastai/learner.py

```diff
@@ -271,8 +271,19 @@ def validate(self, ds_idx=1, dl=None, cbs=None):
         return getattr(self, 'final_record', None)
 
     @delegates(GatherPredsCallback.__init__)
-    def get_preds(self, ds_idx=1, dl=None, with_input=False, with_decoded=False, with_loss=False, act=None,
-                  inner=False, reorder=True, cbs=None, **kwargs):
+    def get_preds(self,
+                  ds_idx: int =1, # Index of data bunch or dataloader
+                  dl: TfmdDL | None=None, # Dataloader to be passed with ds_idx
+                  with_input: bool=False, # Whether to return inputs
+                  with_decoded: bool=False, # Whether to decode based on loss function passed
+                  with_loss : bool=False, # Whether to return losses
+                  act: Any=None, # Option to pass Activation function to predict function
+                  inner : bool=False, # Tells that it's used internally used anywhere like in another training loop
+                  reorder : bool=True, # To order the tensors appropriately
+                  cbs: list | None =None, # Option to pass `Callbacks` to predict function
+                  **kwargs
+                 )-> tuple[tensor, tensor| None]:
```

Comment:
```suggestion
                 )-> tuple:
```

We don't use generic types in fast.ai libs.

Reply:
Noted. Thanks